src/lib.rs

```diff
@@ -92,6 +92,27 @@
 //! }
 //! ```
 //!
+//! # Available logging frameworks
+//!
+//! In order to produce log output executables have to utilize
+//! a framework compatible with the facade.
+//! There are many available logging implementations to chose from.
+//! Here are some of the most popular ones:
+//!
+//! * Simple minimal loggers:
+//!     * [env_logger]
+//!     * [simple_logger]
+//!     * [simplelog]
+//!     * [pretty_env_logger]
```

Comment:
Please remove this from the list for now and file an issue asking the author to add documentation. As is, it doesn't seem intended for people to use.